client/src/common/components/Home.tsx

```diff
@@ -3,164 +3,198 @@ import { Link } from 'react-router-dom';
 import { FaQuestionCircle } from 'react-icons/fa';
 
 import {
-	Accordion,
-	AccordionItem,
-	AccordionItemHeading,
-	AccordionItemButton,
-	AccordionItemPanel,
+  Accordion,
+  AccordionItem,
+  AccordionItemHeading,
+  AccordionItemButton,
+  AccordionItemPanel,
 } from 'react-accessible-accordion';
 import 'react-accessible-accordion/dist/fancy-example.css';
 
 const Home = () => {
-	// test, added style here just to see the classes work
-	return (
-		<div className="bg-primary_bg_color text-primary_title_color font-sans overflow-x-hidden">
-			<section className="p-10 md:p-20 md:grid md:grid-cols-7 md:grid-row-4 grid justify-items-stretch">
-				<h1 className="text-3xl font-bold md:text-7xl mt-5 text-center md:text-left col-start-1 col-span-3">
-					Senpai Kohai
-				</h1>
-				<img
-					className="md:w-6/6 md:row-span-4 md:col-span-4 mt-5"
-					src="./assets/shared_workspace.svg"
-					alt="shared_workspace"
-				/>
-				<p className="mt-5 font-bold text-base w-auto md:text-2xl text-center md:text-left col-span-3">
-					Find your guide, Support youngs
-				</p>
-				<p className="underline cursor-pointer mt-5 w-auto text-sm md:text-lg text-center md:text-left flex col-span-3 justify-self-center md:justify-self-start">
-					<FaQuestionCircle />
-					&nbsp; What are Senpai and Kohai?
-				</p>
-				<div
-					className="cursor-pointer 
+  // test, added style here just to see the classes work
+  return (
+    <main className="bg-primary_bg_color text-primary_title_color font-sans overflow-x-hidden mt-laptopHeaderHeight tablet_md_max:mt-mobileHeaderHeight py-paddingAroundtheContent ">
+      <section className="flex justify-evenly px-laptopPxAroundtheContent tablet_md_max:px-mobileHeaderHeight mb-20">
+        <div className="flex flex-col justify-between tablet_md_max:items-center">
```

Comment:
```suggestion
        <div className="flex flex-col tablet_md_max:items-center tablet_md_max:w-full">
```
![スクリーンショット 2021-12-09 22 34 14](https://user-images.githubusercontent.com/51708229/145634875-be67602d-4c0f-4ca9-be04-839e4619e625.png)

When I check this section on the desktop screen, all items spread like the capture because of `justify-between`. Please remove it.

Also, when I tried to make the "Join Now" button slim the flex container also shrunk together.  To prevent it, added `w-full` for mobile devices.